doc/md/tutorial-todo-gql-schema-generator.md

```diff
@@ -0,0 +1,296 @@
+---
+id: tutorial-todo-gql-schema-generator
+title: Schema Generator
+sidebar_label: Schema Generator
+---
+
+In this section, we continue the [GraphQL example](tutorial-todo-gql.md) by explaining how to generate
+type-safe GraphQL schema from our `ent/schema`.
+
+#### Clone the code (optional)
```

Comment:
is this a new tutorial or another step in the existing one? 
if its continuing the existing one i don't think we need all the house keeping at the top